pmlc/dialect/pxa/ir/ops.td

```diff
@@ -273,6 +273,49 @@ def PxaVectorLoadOp : PxaLoadOpBase<"vector_load"> {
   let hasFolder = 1;
 }
 
+class PxaStoreOpBase<string mnemonic, list<OpTrait> traits = []> :
+    PXA_OpWithPP<mnemonic, !listconcat(traits, [MemRefsNormalizable])> {
+
+  code extraClassDeclarationBase = [{
+    mlir::Value getValueToStore() { return value(); }
+    unsigned getValueOperandIndex() { return 0; }
```

Comment:
These are already generated by the ODS/tablegen, we shouldn't need them.

Reply:
Removed